src/OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper/Logger.cs

```diff
@@ -14,23 +14,23 @@
 // limitations under the License.
 // </copyright>
 
-using OpenTelemetry.AutoInstrumentation.DuckTyping;
+using OpenTelemetry.AutoInstrumentation.Logging;
 
-namespace OpenTelemetry.AutoInstrumentation.Instrumentations.GraphQL;
+namespace OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper;
 
-/// <summary>
-/// GraphQL.ErrorLocation interface for ducktyping
-/// </summary>
-[DuckCopy]
-internal struct ErrorLocationStruct
+internal static class Logger
 {
-    /// <summary>
-    /// Gets the line number of the document where the error occurred
-    /// </summary>
-    public int Line;
+    private static readonly ILogger Log = OtelLogging.GetLogger();
 
-    /// <summary>
-    /// Gets the column number of the document where the error occurred
-    /// </summary>
-    public int Column;
+    internal static void LogTrace(string message)
```

Comment:
It is not a "trace level" log, but a "Information" level log. Can we rename it to `Information` or `LogInformation` or `LogInfo`?